_locales/_untranslated_en.json

```diff
@@ -11,5 +11,14 @@
   },
   "background_sync_disabled_explaination": {
     "message": "Background Sync Disabled, Account is only synced when selected"
+  },
+  "account_muted_description": {
+    "message": "Notifications are disabled for this Account"
```

Comment:
This is a little lengthy, how about just "Muted" or "Notifications muted"?

Reply:
idea was to explain the icon, not sure if "Muted" would be enough? but I guess we could try

Reply:
yip, that would also be my suggestion, just "Muted" is enough. the user has selected "Muted" for the account before and there is an icon, so it is pretty clear